modules/virt-configuring-vm-use-usb-device.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * virt/advanced_vm_management/virt-configuring-usb-host-passthrough.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-configuring-VM-use-usb-device_{context}"]
+= Configuring a virtual machine connection to a USB device
+
+You can configure virtual machine (VM) access to a USB device. This allows a guest to connect to actual USB hardware that is attached to an {product-title} node, as if the hardware and VM are physically connected.
```

Comment:
```suggestion
You can configure virtual machine (VM) access to a USB device. This configuration allows a guest to connect to actual USB hardware that is attached to an {product-title} node, as if the hardware and VM are physically connected.
```

Or something like that. Avoids the ambiguous pronoun.

Reply:
Done!